src/app/modules/customer-management/components/projects/components/services/project.service.spec.ts

```diff
@@ -91,13 +91,38 @@ describe('ProjectService', () => {
     updateProjectRequest.flush(project);
   });
 
+  it('update project using PUT from url locally', () => {
+    const project: Project = { id: '1', name: 'new name', description: 'description', project_type_id: '123', status: 'active'};
+    service.url = 'projects';
+    service.isDevelopment = true;
+    service.updateProject(project).subscribe((response) => {
+      expect(response.name).toBe('new name');
+    });
+    const updateProjectRequest = httpMock.expectOne(`${service.url}/${project.id}`);
+    expect(updateProjectRequest.request.method).toBe('PUT');
+    updateProjectRequest.flush(project);
+  });
+
   it('delete project using DELETE from baseUrl', () => {
     const url = `${service.url}/1`;
+    service.isDevelopment = false;
     service.deleteProject(projectsList[0].id).subscribe((projectsInResponse) => {
       expect(projectsInResponse.filter((project) => project.id !== projectsList[0].id).length).toEqual(2);
     });
     const deleteActivitiesRequest = httpMock.expectOne(url);
     expect(deleteActivitiesRequest.request.method).toBe('DELETE');
     deleteActivitiesRequest.flush(projectsList);
   });
+
+  it('update status project using PUT from baseUrl locally', () => {
+    const url = `${service.url}/1`;
+    service.isDevelopment = true;
+    service.deleteProject(projectsList[0].id).subscribe((projectsInResponse) => {
+      expect(projectsInResponse.filter((project) => project.id !== projectsList[0].id).length).toEqual(2);
+    });
+    const deleteActivitiesRequest = httpMock.expectOne(url);
+    expect(deleteActivitiesRequest.request.method).toBe('PUT');
+    deleteActivitiesRequest.flush(projectsList);
+  });
+
```

Comment:
Try to leave a space between each statement on tests, this should help us to read better.
 
https://github.com/eguezgustavo/tdd_dojo

Like this part:

```
def test__sum_two_numbers_function__returns_twelve__when_inputs_are_five_and_seven():
    #Setup
    number_1 = 5
    number_2 = 7

    #Execution
    result = sum_two_numbers(number_1, number_2)

    #Assertions
    assert result == 12
```